mmv1/third_party/terraform/services/dataproc/resource_dataproc_cluster.go

```diff
@@ -1227,6 +1228,34 @@ func ResourceDataprocCluster() *schema.Resource {
 														},
 													},
 												},
+												"provisioning_model_mix": {
+													Type:     schema.TypeList,
+													Optional: true,
+													AtLeastOneOf: []string{
+														"cluster_config.0.preemptible_worker_config.0.instance_flexibility_policy.0.instance_selection_list",
+														"cluster_config.0.preemptible_worker_config.0.instance_flexibility_policy.0.provisioning_model_mix",
+													},
+													MaxItems:    1,
+													Description: `Defines how Dataproc should create VMs with a mixture of provisioning models.`,
+													Elem: &schema.Resource{
+														Schema: map[string]*schema.Schema{
+															"standard_capacity_base": {
```

Comment:
Do we want to add `ForceNew: true,` as it is not updatable?

Reply:
Yes, seems like I missed it for `provisioning_model_mix` and `standard_capacity_base`.